kotlin/build.gradle

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.50'
+    id 'maven-publish'
+}
+
+/*
+
+TODO: publishing for JVM modules, JS build, and what other platforms?
+
+ */
+
+repositories {
+    mavenCentral()
+}
+
+kotlin {
+
+   /*
+    *  To find out how to configure the targets, please follow the link:
+    *  https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#setting-up-targets
+    */
+
+    jvm {
+      withJava()
+    }
+
+    js('nodejs') {
+        nodejs()
+    }
+
+//    js {
```

Comment:
Why are these commented out? If they're not needed, can you just remove them to keep this file clean?

Reply:
@zongweil the `StringBuilder` method `setCharAt` is not available on all platforms, and I wanted this to be a cross-platform build that others could extend to all targets available for Kotlin.

The commented out section can be removed, I was trying to decide to do a web build or not, and also setup other platforms.  we can remove it and I can add them over time.

Reply:
@zongweil Kotlin can use the Java artifact only for the JVM build but not as a common code base to use across platforms.  And it would be non-standard to publish common without the JVM and JS versions to the maven repo.  I'll double check what is typical in other projects.

Reply:
Sounds good about StringBuilder. I didn't realize that setChatAt was only available for native :).

Please remove the commented out sections for cleanliness for now. It looks simple enough to add back in later.

Yes, if you could point me to another project that publishes Kotlin artifacts to MavenCentral, that would be helpful. It sounds like if someone is writing a Kotlin project and want to use plus codes in their project across all the supported platforms, they would need all the Kotlin artifacts published. Am I understanding that correctly?